README.md

```diff
@@ -15,7 +16,15 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+## My reasoning
+
+The runtime for this program would be $n^2$ because there are nested for loops that each run n times and there is a compare in the inner loop.
```

Comment:
And what is $n$?

Reply:
I have updated it to say that $n$ is the number of vertices in the graph